suggester/src/main/java/org/opengrok/suggest/popular/impl/chronicle/ChronicleMapAdapter.java

```diff
@@ -120,24 +120,29 @@ public void resize(final int newMapSize, final double newMapAvgKey) throws IOExc
 
         Path tempFile = Files.createTempFile("opengrok", "chronicle");
 
-        map.getAll(tempFile.toFile());
+        ChronicleMap<BytesRef, Integer> m;
```

Comment:
If you move assignment below inside the `try` block, then there will be no need for this declaration.